@xen-orchestra/web/src/pages/host/[id]/console.vue

```diff
@@ -23,9 +24,17 @@ const props = defineProps<{
   host: XoHost
 }>()
 
+const { isHostOperationPending } = useHostStore().subscribe()
+
+const STOP_OPERATIONS = [HOST_OPERATION.SHUTDOWN]
```

Comment:
Don't forget to add this host operation in xo-lite too
```suggestion
const STOP_OPERATIONS = [HOST_OPERATION.SHUTDOWN, HOST_OPERATION.REBOOT]
```